ast/misc_test.go

```diff
@@ -127,6 +127,20 @@ load data infile '/tmp/t.csv' into table t fields terminated by 'ab' enclosed by
 	}
 }
 
+// test Change Pump or drainer status sql parser
+func (ts *testMiscSuite) TestChangeStmt(c *C) {
+	sql := `change pump to node_state='paused' for node_id '127.0.0.1:8249';
```

Comment:
Please rewrite the test similar to `TestTableOptimizerHintRestore` to test the Restore() function too.